ocaml/xapi/xapi_globs.ml

```diff
@@ -1255,6 +1258,19 @@ let other_options =
     , (fun () -> !extauth_ad_backend)
     , "Which AD backend used to talk to DC"
     )
+  ; ( "winbind_kerberos_encryption_type"
+    , Arg.String
+        (fun s ->
+          winbind_kerberos_encryption_type :=
```

Comment:
This will set the encryption to Strong if the user get the value wrong. For example if it tries to set it to `lagacy`. While this is safe to do security-wise, it might be surprising to the user.

making `of_string` return an option and using option.iter here would be a noop instead of setting it to default when the value is incorect.

Reply:
We have to make a decision even user input typo, 
will not return an option confuse user?

Reply:
the option is not exposed to the user, see https://github.com/psafont/xen-api/commit/40e81edec73d2c965a739709f640fc200f204ea4#diff-587411f4c101b535ce411cd08905e6001eda94b44c4e76c889b19dbf5f1e8dd6R933